docs/DeveloperGuide.md

```diff
@@ -313,287 +313,289 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
-2. HelloFile recognises the file path to be valid.
-3. HelloFile creates the tag and display the fileInfo.
+1. User requests to tag a file with a tag name, the file path and a label.
+2. HelloFile creates the tag and informs the user.
 
     Use case ends.
 
 **Extensions**
 
-* 2a. HelloFile detects the file path is invalid.
+* 2a. HelloFile detects the tag name entered to be invalid.
 
-    * 2a1. HelloFile prompts the user that the file path is wrong.
+    * 2a1. HelloFile prompts the user that the tag name entered is invalid.
 
         Use case resumes from step 1.
 
-* 2b. HelloFile detects no tag input.
+* 2b. HelloFile detects the file path entered to be invalid.
 
-    * 2b1. HelloFile prompts the user for a tag.
+    * 2b1. HelloFile prompts the user that the file path entered is invalid.
 
         Use case resumes from step 1.
+        
+* 2c. HelloFile detects the label entered to be invalid.
 
-* 2c. HelloFile detects a duplicate tag name.
-
-    * 2c1. HelloFile prompts the user for another tag name.
+    * 2c1. HelloFile prompts the user that the label entered is invalid.
 
         Use case resumes from step 1.
 
-* *a. HelloFile has no permission to read or write to the file system.
+* 2d. HelloFile detects that the tag name already exists in HelloFile.
 
-    * *a1. HelloFile prompts the user that it needs the read and write permission.
+    * 2d1. HelloFile prompts the user that the tag name already exists.
 
-        Use case resumes.
+        Use case resumes from step 1.
 
 <br />
 
+
 **User case ID: UC02**
 
-**Use Case: Access file using tag**
+**Use Case: Open a tagged file**
 
 **MSS**
 
-
-1. User requests to open a file using a tag.
-2. HelloFile recognises the tag to be valid.
-3. HelloFile opens the specified file using the default application.
+1. User requests to open a tagged file.
+2. HelloFile opens the file using the default application.
 
     Use case ends.
 
 **Extension**
 
-* 2a. HelloFile detects that the tag is invalid.
+* 2a. HelloFile cannot find the tag entered.
 
-	* 2a1. HelloFile prompts the user that the tag is wrong.
+	* 2a1. HelloFile prompts the user that the tag cannot be found.
 
         Use case resumes from step 1.
 
-* 3a. HelloFile detects that the target file does not exist.
+* 2b. HelloFile detects that the file path to be invalid.
 
-	* 3a1. HelloFile prompts that the target file does not exist.
+	* 2b1. HelloFile prompts the user that the target file does not exist.
 
         Use case resumes from step 1.
 
-* 3b. HelloFile detects no default application to open the file.
+* 2c. HelloFile detects that there is no permission to open the file.
 
-	* 3b1. HelloFile prompts the user to choose an application to open the file.
+	* 2c1. HelloFile prompts the user that the app does not have the permission to open the file.
 
 	    Use case resumes form step 1.
 
-* 3c. HelloFile encounters an exception when opening the file.
```

Comment:
This should not be deleted?

Reply:
Is this the intended behavior of open command?

Reply:
Okay never mind.